src/main-client.ts

```diff
@@ -194,19 +242,150 @@ export class MainClient extends BaseRestClient {
    *
   **/
 
-  createVirtualSubAccount(params: VirtualSubaccountParams): Promise<any> {
+  createSubAccount(params: CreateSubAccountParams): Promise<VirtualSubAccount> {
     return this.postPrivate('sapi/v1/sub-account/virtualSubAccount', params);
   }
 
-  getSubAccountList(params?: SubAccountListParams): Promise<any> {
+  getSubAccountList(params?: GetSubAccountListParams): Promise<SubAccountListResponse> {
     return this.getPrivate('sapi/v1/sub-account/list', params);
   }
 
-  // TODO: https://binance-docs.github.io/apidocs/spot/en/#sub-account-endpoints
+  getSubAccountSpotAssetTransferHistory(params?: GetSubAccountSpotAssetTransferHistoryParams): Promise<SubAccountSpotAssetTransferHistory> {
```

Comment:
```suggestion
  getSubAccountSpotAssetTransferHistory(params?: SubAccountSpotAssetTransferHistoryParams): Promise<SubAccountSpotAssetTransferHistory> {
```

Tiny nitpick (sorry), would prefer these interfaces to be without the "Get" prefix (following the standard set by the existing getter methods). Same for the other methods where the param interface starts with `Get`